### ISSUE TITLE
Edge update channel doesn't contain definition updates

### ISSUE THREAD

**Issue by kblanks** [cardigann/cardigann#268]
I'm running `1.9.4-33-g667f66b`, but the definition added in commit `667f66b` isn't available, and neither are the definitions added/updated since the last stable release.

**Comment by kblanks** [comment]
![commit 667f66b](https://i.imgur.com/cTfw8fz.png)
![myspleen tracker not in cardigann](https://i.imgur.com/Qd8Vtxb.png)

--

![commit 94a82a8](https://i.imgur.com/FZYbS6v.png)
![filelist test still failing](https://i.imgur.com/vT1oTsA.png)

**Comment by kaso17** [comment]
yes, something is wrong with the build process, @lxt2 has to look at it but he seems to be busy with something else, he hasn't been around lately.

**Comment by kblanks** [comment]
@kaso17 It's been a long time since @lxt2 disappeared, so it would probably be worth re-working the build/release system so new changes can be published.

**Comment by kaso17** [comment]
I've only commit permissions and no access to anything else.
I've no experience with go build systems and I've no intention of becoming the main developer of cardigann (I won't setup the build system).

If you or someone else is willing to take over that role I would help making the necessary changes to point the users to the new home.

**Comment by kblanks** [comment]
I'd happily take over the role, but I have little experience with go. I can put out a call for some people to try for it if you'd be alright with that?

**Comment by kaso17** [comment]
Of course.
I suggest that I create a cardigann2 github account, create a fork, make you admin of the fork. Then you could try getting the build system up and running. Once that's done we could push the necessary changes to the old repo to point to the new one.

In case lxt2 ever comes back I would handover the account details to him.

**Comment by kblanks** [comment]
Creating a new organisation would be better than an actual account (also allows for more fine-tuned permissions). I'd be happy to help out, I know a few Go developers who use Cardigann themselves.

**Comment by kaso17** [comment]
done, you're owner of the org.
Also invited lxt2 as owner

I suggest you create a PR for the old repo with the necessary changes once everything is working.

**Comment by kblanks** [comment]
Thanks, I'll take a look into it now :)

**Comment by kblanks** [comment]
Hey @kaso17, I've managed to compile an updated release with all of the latest commits.

![ToTheGlory appears in Cardigann](https://i.imgur.com/5pOwePx.png)
![The recent commit that adds ToTheGlory](https://i.imgur.com/qJXU7fW.png)

The next step is to migrate away from the equinox build system (we can just use GitHub Releases, which is free) and modify the code to point to the new repository. I'll work on this now :)

**Comment by raspdealer** [comment]
any updates?

**Comment by kaso17** [comment]
@kailan made any progress/still interested? Otherwise I would put up a "looking for new maintainer" note in the readme.

**Comment by kailan** [comment]
I have a lot of free time now, so I can continue working on cardigann2 in
the next few days :)
